packages/typespec-azure-portal-core/lib/decorators.tsp

```diff
@@ -15,6 +15,13 @@ extern dec about(target: Model, options: AboutOptions);
  */
 extern dec browse(target: Model, options: BrowseOptions);
 
+/**
+ * Provides a Model customizing deployment promotion apiVersion for ARM resource.
+ * The apiVersion will be used as a version to deploy to Portal.
+ * @param options Property options provides promotion information of the resourceType.
+ */
+extern dec promotion(target: Model, options: PromotionOptions);
```

Comment:
Do you plan to always take a single version or an array?

Reply:
only single version always :)